vis-lua.c

```diff
@@ -2038,7 +2039,8 @@ static int window_style(lua_State *L) {
 	enum UiStyle style = luaL_checkunsigned(L, 2);
 	size_t start = checkpos(L, 3);
 	size_t end = checkpos(L, 4);
-	win_style(win, style, start, end);
+	bool keep = lua_isboolean(L, 5) && lua_toboolean(L, 5);
```

Comment:
```suggestion
	if (!lua_isboolean(L, 5))
		return luaL_argerror(L, 5, "boolean expected");
```
I think we should raise an error rather than assume false when the caller passes something weird.

Reply:
All other functions use `lua_isboolean(L, 5) && lua_toboolean(L, 5)` approach.
If we raise an error in this function it would be appropriate to do the same in all of the others.